src/test/java/Homework17.java

```diff
@@ -0,0 +1,76 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework17 extends BaseTest {
+    @Test
+    public void addSongToPlaylist() {
+        openLoginUrl();
+        enterEmail("demo@class.com");
+        enterPassword("te$t$tudent");
+        clickSubmit();
+        searchForSong("Dark Days");
+        viewAllSongs();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        verifySearchUrl();
+        selectFirstSearchResult();
+        clickAddToButton();
+        choosePlaylist();
+
+
+        //finds View All button
+        WebElement viewAllSongs = driver.findElement(By.cssSelector("data-test='view-all-songs-btn"));
+        viewAllSongs.click();
+        //finds first result in the View All list
+        WebElement firstResult = driver.findElement(By.xpath("//tr[@class='song-item']//td[contains(text(), \"Dark Days\")]"));
+        firstResult.click();
+
+        //clicks the 'Add to' button
+        WebElement addTo = driver.findElement(By.xpath("//button[@class='btn-add-to']"));
+        addTo.click();
+
+    }
+
+    private void choosePlaylist() {
+        WebElement testPlaylist = driver.findElement(By.xpath("//a[contains(text(),'test')]"));
+        testPlaylist.click();
+
+        //verifies the addition of the song to the playlist
+        WebElement addedToPlaylistMessage = driver.findElement(By.xpath("//div[@class='success']"));
```

Comment:
this selector wrong -> check and correct it
//HINT: WebElement addedToPlaylistMessage = driver.findElement(By.xpath("//div[@class='success show']"));